recipes/openvino-telemetry/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "openvino-telemetry" %}
+{% set version = "2022.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 41eec1f6a9bbb528971bea8cf545ab9627e80e67522345fe77d2df3a2d3db73e
+
+build:
+  number: 0
+  entry_points:
+    - opt_in_out = openvino_telemetry.opt_in_out:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - requests >=2.20.0
```

Comment:
You should not need the run time dependency at build time but it seems that this package imports itself at build time :-/

PS: can you open an issue upstream about this? It is bad practice to require a local installation to be able to build a package.